nengo/builder/tests/test_transforms_conv.py

```diff
@@ -14,8 +14,9 @@
 @pytest.mark.parametrize("kernel0", (4, 5))
 @pytest.mark.parametrize("kernel1", (4, 5))
 @pytest.mark.parametrize("padding", ("same", "valid"))
+@pytest.mark.parametrize("groups",  (1, 8, 32))
```

Comment:
There are already a lot of parametrizations for this test, and adding "groups" makes it go from 64 to 192. I know they're quite fast, but it adds clutter to the log and I'm not convinced it's necessary. Also, it might be good to test with different numbers of in and out channels, since that's directly related to groups.

The way I would do this is to take the body of this function, and make a `_test_convinc_2d` helper function. Then you can have two functions, `test_convinc_2d` and `test_grouped_convinc_2d`, that test the ungrouped and grouped cases. Since we're splitting up `conv2d` into grouped and ungrouped versions, this will also allow testing each one separately.